tap_process_street/schema.py

```diff
@@ -1,59 +1,29 @@
-import os
 import json
-from singer import metadata
-
-from tap_process_street.constant import ID_KEY, UPDATED_DATE_KEY
-
-STREAMS = {
-    'checklists': {
-        'key_properties': [ID_KEY],
-        'replication_method': 'INCREMENTAL',
-        'replication_keys': [UPDATED_DATE_KEY, ID_KEY]
-    },
-    'form-field-values': {
-        'key_properties': [ID_KEY],
-        'replication_method': 'INCREMENTAL',
-        'replication_keys': [UPDATED_DATE_KEY, ID_KEY]
-    },
-    'form-field-widgets': {
-        'key_properties': [ID_KEY],
-        'replication_method': 'INCREMENTAL',
-        'replication_keys': [UPDATED_DATE_KEY, ID_KEY]
-    },
-    'tasks': {
-        'key_properties': [ID_KEY],
-        'replication_method': 'INCREMENTAL',
-        'replication_keys': [UPDATED_DATE_KEY, ID_KEY]
-    },
-    'templates': {
-        'key_properties': [ID_KEY],
-        'replication_method': 'INCREMENTAL',
-        'replication_keys': [UPDATED_DATE_KEY, ID_KEY]
-    }
-}
+import os
+
+STREAMS = [
+    'checklists',
+    'form-field-values',
+    'form-field-widgets',
+    'tasks',
+    'templates',
+    'users'
+]
+
```

Comment:
The keys and replication method are always the same so we don't need to duplicate it.